system/library/razorpay-sdk/libs/Requests-1.8.0/library/Requests/Transport/cURL.php

```diff
@@ -150,7 +150,7 @@ public function request($url, $headers = array(), $data = array(), $options = ar
 		if (isset($options['verify'])) {
 			if ($options['verify'] === false) {
 				curl_setopt($this->handle, CURLOPT_SSL_VERIFYHOST, 0);
-				curl_setopt($this->handle, CURLOPT_SSL_VERIFYPEER, 0);
+				curl_setopt($this->handle, CURLOPT_SSL_VERIFYPEER, 1);
```

Comment:
@jampulanaveen , please replace this file from SDK again. As it's hardcoded here. Apply same in all the open PR's.